Try to fix:
posixovl.c:196:14: error: conflicting types for ‘strlcpy’
 static char *strlcpy(char *dest, const char *src, size_t n)
              ^~~~~~~
In file included from posixovl.c:33:
/usr/include/string.h:84:8: note: previous declaration of ‘strlcpy’ was here
 size_t strlcpy (char *, const char *, size_t);
        ^~~~~~~
make[1]: *** [Makefile:446: posixovl.o] Error 1


--- posixovl-1.3/posixovl.c.orig	2018-09-20 08:41:18.000000000 +0000
+++ posixovl-1.3/posixovl.c	2019-09-29 19:17:13.000000000 +0000
@@ -193,13 +193,6 @@
 	return in + 1;
 }
 
-static char *strlcpy(char *dest, const char *src, size_t n)
-{
-	strncpy(dest, src, n);
-	dest[n-1] = '\0';
-	return dest;
-}
-
 /**
  * __hl_dtoi - build the HL.I-node path from the HL.D-node path
  */
